modules/standard/Sort.chpl

```diff
@@ -559,38 +595,13 @@ proc sort(ref Data: [?Dom] ?eltType, comparator:?rec=defaultComparator,
           param stable:bool = false, param inPlaceAlgorithm:bool = false) {
   chpl_check_comparator(comparator, eltType);
 
-  if Dom.low >= Dom.high then
-    return;
```

Comment:
IMO it's nicer to have this check in both a future `stableSort` as well as `unstableSort` rather than having it in all the places that call these. Anyway I'll add a TODO comment so we remember to add it to `stableSort`.